cmd/thor/solo/solo.go

```diff
@@ -87,6 +91,10 @@ func (s *Solo) Run(ctx context.Context) error {
 }
 
 func (s *Solo) loop(ctx context.Context) {
+	if err := s.initSolo(); err != nil {
+		log.Error("failed to add first block txs", "err", err)
```

Comment:
Do we want to terminate solo here if there is any error or just log it and continue without the gas tx?

Reply:
also can we add some integration test for it if possible ?

Reply:
@paologalligit I changed it to panic.

@leszek-vechain I added a test